### DISCUSSION TITLE
More sensors in simulator

### DISCUSSION THREAD

**Discussion by kartun83** [zepp-health/zeppos-forum#252] [Developer Feedback]
It would be very handy to have all available sensors in simulator for watchface debugging.
Currently there are:

- heart rate
- calories
- steps
there is no data for:

- pai
- distance
- weather
- temperature
- oxygen
- etc

also there would be very useful toggle for BT connection

**Comment by MyoungXUE** [comment]
Hi, that's a very good suggestion, the Zepp OS team is planning to enhance the simulator  in the future to include a data mock, stay tuned!
